app/models/model.rb

```diff
@@ -0,0 +1,4 @@
+#Placeholder for a model
+
+class Post < ActiveRecord::Base
```

Comment:
remove any of this code. Just leave the file. I want them to remember that to get the magical methdos they must inherit from `ActiveRecord::Base`